modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaClientCodegen.java

```diff
@@ -244,6 +243,7 @@ public void processOpts() {
             // the "okhttp-gson" library template requires "ApiCallback.mustache" for async call
             supportingFiles.add(new SupportingFile("ApiCallback.mustache", invokerFolder, "ApiCallback.java"));
             supportingFiles.add(new SupportingFile("ApiResponse.mustache", invokerFolder, "ApiResponse.java"));
+            supportingFiles.add(new SupportingFile("JSON.mustache", invokerFolder, "JSON.java"));
```

Comment:
The `JSON.mustache` template is required by the `jersey2` library as well. Currently there would be a compilation error of "can not find symbol (JSON)" if cleaning and regenerating the jersey2 petstore sample:

```
mvn clean package

rm -rf samples/client/petstore/java/jersey2
git checkout samples/client/petstore/java/jersey2/src/test
./bin/java-petstore-jersey2.sh

cd samples/client/petstore/java/jersey2
mvn test
```